underfs/hdfs/src/main/java/alluxio/underfs/hdfs/acl/SupportedHdfsAclProvider.java

```diff
@@ -67,18 +71,70 @@ public AccessControlList getAcl(FileSystem hdfs, String path) throws IOException
     return acl;
   }
 
-  private AclEntryType getAclEntryType(AclEntry entry) throws IOException {
+  @Override
+  public void setAcl(FileSystem hdfs, String path, AccessControlList acl) throws IOException {
+    // convert AccessControlList into hdfsAcl
+    List<AclEntry> aclSpecs = new ArrayList<>();
+
+    for (alluxio.security.authorization.AclEntry entry : acl.getEntries()) {
+      AclEntry hdfsAclEntry = getHdfsAclEntry(entry);
+      aclSpecs.add(hdfsAclEntry);
+    }
+    // set hdfsAcl;
+    hdfs.setAcl(new Path(path), aclSpecs);
+    // set owner and owning group
+    hdfs.setOwner(new Path(path), acl.getOwningUser(), acl.getOwningGroup());
```

Comment:
is this necessary? I don't think we can change the owner or group with the setacl command?

Reply:
removed